hoodie-client/src/main/java/com/uber/hoodie/io/storage/HoodieWrapperFileSystem.java

```diff
@@ -536,8 +536,9 @@ public void completeLocalOutput(Path fsOutputFile, Path tmpLocalFile) throws IOE
   }
 
   @Override
-  public void close() throws IOException {
-    fileSystem.close();
+  public void close() {
```

Comment:
Are we depending on the JVM shutdown to automatically close() or should we register it in the hook ?

Reply:
we need shutdown hook only if we open up sockets etc here.. since this just wraps another filesystem. we dont need to do anything special I think

Reply:
Well, ideally, you might not notice any difference now but ideally we want to call the fs.close() in your finally block if you are using the raw file system, so we should have some code clean up the fs connection and not depend on jvm shutdown.

@vinothchandar